usaspending_api/disaster/v2/views/agency/spending.py

```diff
@@ -0,0 +1,133 @@
+            # Currently, this endpoint can never have children.
+            "children": Value([], output_field=ArrayField(IntegerField())),
+            "count": Value(0, output_field=IntegerField()),
+            "obligation": Coalesce(
+                Sum(
+                    Case(
+                        When(
+                            self.final_period_submission_query_filters,
+                            then=F("obligations_incurred_by_program_object_class_cpe"),
+                        ),
+                        default=Value(0),
+                    )
+                ),
+                0,
+            ),
+            "outlay": Coalesce(
+                Sum(
+                    Case(
+                        When(
+                            self.final_period_submission_query_filters,
+                            then=F("gross_outlay_amount_by_program_object_class_cpe"),
+                        ),
+                        default=Value(0),
+                    )
+                ),
+                0,
+            ),
+            "total_budgetary_resources": Coalesce(
+                Sum("treasury_account__gtas__budget_authority_appropriation_amount_cpe"), 0
+            ),
+        }
+
+        return (
+            FinancialAccountsByProgramActivityObjectClass.objects.filter(*filters)
+            .values(
+                "treasury_account__funding_toptier_agency",
+                "treasury_account__funding_toptier_agency__toptier_code",
+                "treasury_account__funding_toptier_agency__name",
+            )
+            .annotate(**annotations)
+            .values(*annotations.keys())
+        )
+
+    @property
+    def award_queryset(self):
+        filters = [
+            Q(disaster_emergency_fund__in=self.def_codes),
+            Q(treasury_account__isnull=False),
+            Q(treasury_account__funding_toptier_agency__isnull=False),
+            self.all_closed_defc_submissions,
+        ]
+
+        annotations = {
+            "id": F("treasury_account__funding_toptier_agency"),
+            "code": F("treasury_account__funding_toptier_agency__toptier_code"),
+            "description": F("treasury_account__funding_toptier_agency__name"),
+            # Currently, this endpoint can never have children.
+            "children": Value([], output_field=ArrayField(IntegerField())),
+            "count": Value(0, output_field=IntegerField()),
+            "obligation": Coalesce(Sum("transaction_obligated_amount"), 0),
+            "outlay": Coalesce(
+                Sum(
+                    Case(
+                        When(self.final_period_submission_query_filters, then=F("gross_outlay_amount_by_award_cpe")),
+                        default=Value(0),
+                    )
+                ),
+                0,
+            ),
+            "total_budgetary_resources": Value(None, DecimalField()),  # NULL for award spending
```

Comment:
The object class PR: https://github.com/fedspendingtransparency/usaspending-api/pull/2584 chooses to strip this form the response altogether. Might be better to just merge now and refactor later, but we should aim for consistency in the future.